R/uvhydrograph-data.R

```diff
@@ -22,7 +22,11 @@ parsePrimaryUVData <- function(data, month, useDownsampled=FALSE) {
   est_UV <- subsetByMonth(getTimeSeries(data, timeSeriesNames$primarySeriesName, estimatedOnly=TRUE), month)
   uncorr_UV <- subsetByMonth(getTimeSeries(data, timeSeriesNames$primarySeriesRawName ), month)
   comp_UV <- subsetByMonth(getTimeSeries(data, timeSeriesNames$comparisonSeriesName ), month)
-  water_qual <- subsetByMonth(getWaterQualityMeasurements(data), month)
+  water_qual <- tryCatch({
+    subsetByMonth(readWaterQualityMeasurements(data), month)
```

Comment:
would you want `subsetByMonth` to occur outside of the tryCatch for `readWaterQualityMeasurements`? I guess it depends on if you want the code to actually stop if there's an error with subsetting or just return an NA data frame.